client/src/components/PhotoModal.jsx

```diff
@@ -2,102 +2,55 @@ import React from 'react';
 import Photo from './Photo.jsx';
 import PhotoModalCarousel from './PhotoModalCarousel.jsx';
 import PhotoModalDescriptionContainer from './PhotoModalDescriptionContainer.jsx';
-import { FaChevronRight, FaChevronLeft, FaTimes} from 'react-icons/fa';
-
-class PhotoModal extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      currentPhoto: 0,
-    };
-    this.handleRightArrowClick = this.handleRightArrowClick.bind(this);
-    this.handleLeftArrowClick = this.handleLeftArrowClick.bind(this);
-  }
-
-  handleLeftArrowClick() {
-    const { currentPhoto } = this.state;
-    const { photos } = this.props;
-    const newCurrentPhoto =
-      currentPhoto > 0 ? currentPhoto - 1 : photos.length - 1;
-
-    this.setState({
-      currentPhoto: newCurrentPhoto,
-    });
-  }
-
-  handleRightArrowClick() {
-    const { currentPhoto } = this.state;
-    const { photos } = this.props;
-    const newCurrentPhoto =
-      currentPhoto < photos.length - 1 ? currentPhoto + 1 : 0;
-
-    this.setState({
-      currentPhoto: newCurrentPhoto,
-    });
-  }
-
-  render() {
-    const { photos } = this.props;
-    const { currentPhoto } = this.state;
-    const { viewPhotoButtonClick } = this.props;
-
-    return (
-      <div>
-        <div className="photoModuleParentContainer">
-          <div className="photoModuleLeftContainer">
-            <div className="photoModuleLeftArrowContainer">
-              <div className="photoModuleLeftArrow">
-                <span className="FaChevronLeft" onClick={this.handleLeftArrowClick}>
-                  <FaChevronLeft />
-                </span>
-              </div>
-
-            </div>
-
-            <div className="photoModuleMainPhotoContainer">
-              <div className="photoModuleMainPhoto">
-                <Photo photoClass={'modalPhoto'} photoUrl={photos[currentPhoto]}/>
-
-              </div>
-
+import { FaChevronRight, FaChevronLeft, FaTimes } from 'react-icons/fa';
+import '../../dist/PhotoModal.css';
+
+const PhotoModal = ({ currentPhoto, handleLeftArrowClick, handleRightArrowClick, photos, viewPhotoButtonClick }) => {
+
+  return (
```

Comment:
Make this an implicit return (arrow function). Just use parentheses with no return keyword.

```
const someComponent = () => (
  <div>
    <otherThing />
  </div>
);
```